Observer/CustomerSaveAfter.php

```diff
@@ -85,27 +85,24 @@ public function execute(Observer $observer)
                 1,
                 true
             );
-            if ($isCustomerSyncActive) {
+
+            $isCheckoutInProgress = $this->request->getParam('_checkout_in_progress', null);
+            if ($isCustomerSyncActive && !$isCheckoutInProgress) {
```

Comment:
Because in this case, there's only a `if a === null do:` , and no else clause. Which is the same and could be replaced with `not $a do:`. I reverted it for not changing the logic.